src/app/models/relationship.rb

```diff
@@ -0,0 +1,11 @@
+class Relationship < ApplicationRecord
```

Comment:
we'll need to be more specific with what `relationship` is it? because it's too broad of a name, in our case it's `followership`